content/blog/2018/01/2018-01-26-fosdem-hackaton.adoc

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "You're invited to the Post-FOSDEM 2018 Jenkins Hackfest"
+tags:
+- fosdem
+- event
+author: lnewman
+---
+
+On the first weekend in February numerous free and open source developers from around the
+world will travel to Brussels, Belgium, for arguably the largest event of its kind:
+link:https://fosdem.org[FOSDEM].
+Among the thousands of hackers in attendance will be a number of Jenkins contributors.
+
+On the Monday after FOSDEM,
+a group of those contributors will get together to spend a full day hacking on Jenkins and *you* are
+link:https://www.meetup.com/jenkinsmeetup/events/246098584/[invited to join them].
+Folks of all experience levels are welcome,
+there will be sessions for everyone from seasoned hackers to new contributors.
+
+== All-day Jenkins Hackfest
+
+The Hackfest will start at 9:30am on Monday with general introductions and gathering of potential topics/projects.
+Everyone is encouraged to bring topic suggestions that they'd like to work on, but if you don't have one there will be plenty from which to choose.
```

Comment:
This seems a little awkward -- not sure of better wording but here are some possibilities:

Everyone is encouraged to bring topic suggestions but this is not required; there will be plenty from which to choose.

You can bring a list of topics on which you'd like to work but this is not required; there will be plenty from which to choose.

Bring suggestions of topics that interest you if you like, or just come and choose from the topics others suggest.

I don't know -- none of these are great and they all change the sense of what you're saying...  So maybe you should just stick with what you have.